Frontend/README.md

```diff
@@ -0,0 +1,51 @@
+# graphql-api-base-frontend
+
+A frontend for `graphql-api-base`.
+
+## Getting started
+
+Install
+
+```bash
```

Comment:
I personally prefer inline to make it shorter. (Also in bold)
**Install:** `yarn`

Reply:
Nice